frontend/src/components/Displays/LocalizationVerification/ScheduleMissionChecklistDialog.tsx

```diff
@@ -0,0 +1,256 @@
+            robot.pressureLevel < robot.model.upperPressureWarningThreshold
+        ) {
+            statusText = `${TranslateText('The current pressure level is')} ${robot.pressureLevel} ${TranslateText(
+                'which is within the suggested range'
+            )}`
+        } else {
+            statusText = `${TranslateText('Warning')}: ${TranslateText('The current pressure level is')} ${
+                robot.pressureLevel
+            } ${TranslateText('which is NOT within the suggested range')}`
+        }
+    } else {
+        statusText = TranslateText(
+            'The pressure measurement is not currently available, so start the mission at your own risk.'
+        )
+    }
+
+    return (
+        <>
+            <Dialog.Header>
+                <Typography variant="h5">{TranslateText('Confirm pressure level of robot')}</Typography>
+            </Dialog.Header>
+            <Dialog.Content>
+                <VerticalContent>
+                    <Typography>
+                        {`${robot.name} (${robot.model.type}) ${TranslateText(
+                            'needs to be have a pressure level of between'
+                        )} ${robot.model.lowerPressureWarningThreshold} 
+                        ${TranslateText('and')} ${robot.model.upperPressureWarningThreshold} `}
+                        <b>{TranslateText('before')}</b>
+                        {` ${TranslateText('clicking confirm')}. `}
+                        {statusText}
+                    </Typography>
+                    <HorizontalContent>
+                        <Checkbox
+                            crossOrigin={undefined}
+                            onChange={(e: ChangeEvent<HTMLInputElement>) => setIsCheckConfirmed(e.target.checked)}
+                        />
+                        <Typography>
+                            {`${TranslateText('I confirm that')} ${robot.name} (${robot.model.type}) ${TranslateText(
+                                'has a pressure level making it safe for the area it is operating in'
+                            )} `}
+                            <b>{deckName}</b>
+                        </Typography>
+                    </HorizontalContent>
+                </VerticalContent>
+            </Dialog.Content>
+        </>
+    )
+}
+
+const BatteryDialog = ({ setIsCheckConfirmed, robot, deckName }: CheckDialogProps) => {
+    const { TranslateText } = useLanguageContext()
+
+    let statusText = ''
+    if (robot.batteryLevel) {
+        if (!robot.model.batteryWarningThreshold || robot.batteryLevel > robot.model.batteryWarningThreshold) {
+            statusText = `${TranslateText('The current battery level is')} ${robot.pressureLevel}
+                            ${TranslateText('which is above the suggested minimum')}`
+        } else {
+            statusText = `${TranslateText('Warning')}: ${TranslateText('The current battery level is')} ${
+                robot.pressureLevel
```

Comment:
![image](https://github.com/equinor/flotilla/assets/108514046/271fed29-67a3-402f-b418-96a00b8e88d3)

battery level is being compared with robot.pressure